applications/MeshingApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -25,7 +25,9 @@
 
 #include "custom_utilities/local_refine_triangle_mesh.hpp"
 #include "custom_utilities/local_refine_prism_mesh.hpp"
+#ifdef  STRUCTURAL_MECHANICS_APPLICATION
```

Comment:
why is this needed?

Reply:
note that you should REALLY AVOID cross including this file

Reply:
I can move it to the core or reverting to only header file

Reply:
Or I can remove the utility, which with MMG is deprecated